app/helpers/assignment_result_helper.rb

```diff
@@ -25,7 +26,7 @@ def report_bug_link(assignment, organization=Organization.current)
         fa_icon(:bug, text: t(:notify_problem_with_exercise), class: 'fa-fw'),
         subject: t(:problem_with_exercise, title: @exercise.name),
         body: assignment_help_email_body(assignment),
-        class: 'warning'
+        class: 'dropdown-item'
```

Comment:
I does not understand this change

Reply:
This is because all dropdown items need that class for correct styling.

Why not `class: 'warning dropdown-item'`? The `warning` class was doing absolutely nothing, at least on my tests. There is no CSS definition for `warning` nor we were referencing it via JS or anything.

Maybe I didn't see it but I'm pretty sure it's useless.

Reply:
Great!